### DISCUSSION TITLE
Incorrect default owner selection when creating an item in a collection

### DISCUSSION THREAD

**Discussion by florian-bellencontre** [dani-garcia/vaultwarden#3465] [Q&A]
### Subject of the issue
I don't know if it's a Bitwarden or Vaultwarden bug but if in doubt I post it here, because I couldn't test it on Bitwarden.

The problem is a bad selection of the default owner when creating an item in a collection. This is more an enhancement request than a bug to improve the final user experience.

### Deployment environment
Diag:
![Screenshot from 2023-04-19 14-11-24](https://user-images.githubusercontent.com/57491181/233071058-34259441-8101-4541-bf3f-5fa9d6c370f2.png)

* vaultwarden version: 1.28.1
* Install method: Installation from releases and with a compilation
* Clients used: Browser (firefox, chrome)
* Reverse proxy and version: haproxy

### Steps to reproduce
1. Go to the Vaultwarden home page
2. Check that we are in "All vaults".
3. Select a collection
4. Create an item in this collection

### Expected behaviour
It would be better if the selected owner is the organization and the collection where I am is selected.

Kind of like if I did this:
1. Go to the Vaultwarden home page
2. Check that we are in my needed organization "Outsourcing" at the top
3. Select the collection
4. Create an item in the collection

We can see here that the owner is the organization and that the right collection is selected

Screenshot:
![Screenshot from 2023-04-19 14-21-25](https://user-images.githubusercontent.com/57491181/233078484-c1878206-47c2-4656-ad2b-a8df10162e1b.png)
![Screenshot from 2023-04-19 14-21-42](https://user-images.githubusercontent.com/57491181/233078516-a88524b4-814e-4ab1-9bec-3ca3321d8f74.png)

### Actual behaviour
The default owner is my user "noc..":
![Screenshot from 2023-04-19 14-20-31](https://user-images.githubusercontent.com/57491181/233077346-32d6d91d-c56d-40ce-8c64-02e12c44ba37.png)

![Screenshot from 2023-04-19 14-20-56](https://user-images.githubusercontent.com/57491181/233077392-21b4d309-93df-4405-9d41-e7c196c06e71.png)

**Comment by BlackDex** [comment]
This is a client side item. Which is a Bitwarden maintained item.
Maybe there already is a feature request regarding this else i suggest to create one here https://community.bitwarden.com/c/feature-requests/5/none
